src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForIteratorPadCheck.java

```diff
@@ -57,6 +57,18 @@
  *   &lt;property name=&quot;option&quot; value=&quot;space&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>violation</p>
+ * <pre>
+ * for(int i; i&lt;10;){
+ *     //violation
```

Comment:
1. should be `// violation`
2. should be placed on line with violation and describe what violation is, here it is not clear